test/e2e/miragevpn-server.sh

```diff
@@ -28,9 +28,9 @@ trap cleanup EXIT
 
 (
 	sleep 1
-	openvpn --cd "$config_dir" --config "client.conf" --dev-type tun --dev "$tun_interface" --writepid "$pidfile" --script-security 2 --up ../client-up.sh > /dev/null
+	openvpn --cd "$config_dir" --config "client.conf" --dev-type tun --dev "$tun_interface" --writepid "$pidfile" --script-security 2 --up ../client-up.sh
```

Comment:
I'm undecided about the output here.. somehow it is nice to spot the issues easily in CI, somehow it is also nice to not get a lot of output.

In the end, I'm fine without the redirection to /dev/null. if there's need to see less output, someone invoking the scripts can still redirect to /dev/null. :)

Reply:
I removed the redirection to /dev/null to debug in CI. We should add it back, I just forgot.